tests/layer_tests/ovc_python_api_tests/test_tf.py

```diff
@@ -49,7 +53,7 @@ def create_keras_model(temp_dir):
     x1 = tf.keras.Input(shape=input_shape, name=input_names[0])
     x2 = tf.keras.Input(shape=input_shape, name=input_names[1])
     y = tf.nn.sigmoid(tf.nn.relu(x1 + x2))
-    keras_net = tf.keras.Model(inputs=[x1, x2], outputs=[y])
+    keras_net = tf.keras.Model(inputs=[x1, x2], outputs=[{"output": y}])
```

Comment:
why is this changed? We should also support output list.

Reply:
This change is not necessary. It was done only to reuse Keras model in new tests, which check output names and need named outputs.  
I removed this change and added another method that creates Keras model with named outputs.